Shadowlands/Plaguefall/Trash.lua

```diff
@@ -181,6 +183,14 @@ function mod:BelchPlague(args)
 	end
 end
 
+function mod:BeckonSlime(args)
+	local unit = self:GetUnitIdByGUID(args.sourceGUID)
+	if unit and UnitAffectingCombat(unit) then
```

Comment:
These mobs sometimes cast Belch Plague outside of combat, that's why this check is in its handler. Unless it's the same with Beckon Slime, there's no need for this.